docs/application/native/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,642 @@
+            <icon>org.tizen.sample.png</icon>
+            <label>base-frame</label>
+            <label xml:lang="en-us">base-frame</label>
+            <label xml:lang="ko-kr">base-frame[KOR]</label>
+        </frame-component>
+    </component-based-application>
+    <privileges>
+        <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+    </privileges>
+</manifest>
+
+```
+
+Pay specific attention to the following attributes:
+
+- `id`
+
+  Every component element has id attribute. The id has to be globally unique in device. If not, application installation would be failed. And this attribute will be used by other applications to tell which component should be created.
+
+- `main`
+
+  The Component-Based Applications are able to have multiple components and there is one main component. The main component is the component that created when the Component-Based Application received launch request without specific component ID. You can set main component by set this attribute "true"
+
+
+## Prerequisites
+
+To use the functions and data types of the Component-Based Application API (in mobile and wearable applications), include the `<component_based_app.h>` header file in your application:
+
+```
+#include <component_based_app.h>
+```
+
+<a name="application_monitoring"></a>
+## Run Component-Based Application
+
+To run a Component-Based Application, you should register the Component-Based Application callbacks and start a main event loop as following codes:
+
+```
+component_class_h __app_create_cb(void *user_data)
+{
+     component_class_h comp_class = NULL;
+    return comp_class;
+}
+
+void __app_terminate_cb(void *user_data)
+{
+}
+
+int main(int argc, char** argv)
+{
+    int ret;
+    component_based_app_lifecycle_callback_s callback = {
+        .create = __app_create_cb,
+        .terminate = __app_terminate_cb
+    };
+
+    ret = component_based_app_main(argc, argv, &callback, NULL);
+    if (ret != APP_ERROR_NONE)
+        return ret;
+
+    return 0;
+}
+```
+
+<a name="frame_component_monitoring"></a>
+## Managing Frame-Component
+
+To add and manage Frame-Component, you should add Frame-Component and registering Frame-Component callbacks.
+
+1. Declaring a Frame-Component in the manifest file
+    ```
+    <component-based-application appid="org.tizen.base-component" exec="@BINDIR@/base-component" nodisplay="false" multiple="false" type="capp">
+        <label>Base-component Application</label>
+        <icon>@DESKTOP_ICON@</icon>
+        <frame-component id="base-frame" launch_mode="caller" main="true" icon-display="false" taskmanage="true">
+            <icon>org.tizen.sample.png</icon>
+            <label>base-frame</label>
+            <label xml:lang="en-us">base-frame</label>
+            <label xml:lang="ko-kr">base-frame[KOR]</label>
+        </frame-component>
+    </component-based-application>
+    ```
+
+2. Adding a Frame-Component to the Component-Based Application
```

Comment:
As there is a code block followed by the sentence, please use a ":" at the end of the sentence.
Global comment:
In other guides, the gerund form is not used for steps.
So, can we rephrase to:
Add a Frame-Component to the Component-Based Application:

Please consider lines 561, 418, 433, 463, 527, and 600.

Reply:
Done